org.coolercontrol.CoolerControl.yml

```
@@ -0,0 +1,87 @@
+id: org.coolercontrol.CoolerControl
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+separate-locales: false
+command: coolercontrol
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
```

so there is no permission to display anything at all.

---

The permissions are sufficient for the application to work and the X11-fallback works testing locally, due to the Tauri bug with wayland. Do you mean that there's something missing here?

---

```suggestion
  - --socket=x11
```

the linter will prevent it anyway.